### DISCUSSION TITLE
Building Vernier with OpenMPI

### DISCUSSION THREAD

**Discussion by arporter** [MetOffice/Vernier#154] [General]
Hello,

I see the docs say that MPICH is required. Is that really true or should any implementation of MPI work? I ask because I'm trying with OpenMPI but having no joy (CMake says that various MPI dependencies are missing).

Thanks :-)

Andy.

**Comment by hiker** [comment]
I used (installing it in a temporary 'local' directory in vernier itself):
```
cmake -DCMAKE_INSTALL_PREFIX:PATH=$PWD/../local -DENABLE_MPI=off  -DBUILD_TESTS=off ..
```
and it worked for me without MPI. I had to disable testing, since I ran into problems with google testing (which reported time outs, see #149)

**Comment by arporter** [comment]
Thanks Joerg, that did the trick :-)
